src/coreclr/vm/callhelpers.cpp

```diff
@@ -545,9 +545,7 @@ void MethodDescCallSite::CallTargetWorker(const ARG_SLOT *pArguments, ARG_SLOT *
     if (transitionToPreemptive)
     {
         GCPreemp transitionIfILStub(transitionToPreemptive);
-        DWORD* pLastError = &GetThread()->m_dwLastErrorInterp;
```

Comment:
The interpreter support for last error is broken (it was not updated for source generated stubs) and this change breaks it even more. I am deleting it instead of trying to do some sort of quick fix.